R/commonTTest.R

```diff
@@ -125,13 +125,13 @@
 }
 
 .ttestDescriptivesContainer <- function(jaspResults, options) {
-  if(!options$descriptives && !options$descriptivesPlots && !options$descriptivesPlotsRainCloud) return()
+  if(!options$descriptives && !options$descriptivesPlots && !options$descriptivesPlotsRainCloud && !options$descriptivesPlotsRainCloudDifference)
```

Comment:
This gives me the following error in the classical one-sample t-test:
![Screenshot 2020-12-04 at 21 45 53](https://user-images.githubusercontent.com/7093546/101213106-26374b00-367a-11eb-8ee7-5a156e8f1353.png)

Perhaps you want to use `isFALSE()` here

Reply:
Yes, that solves the issue! I don't know why it didn't give me an error with the unit tests before...